docs/api-docs/payments/payments-api-overview.mdx

```diff
@@ -387,7 +438,7 @@ The Payments API allows developers to store a credit card while processing a pay
 
 When processing a credit card payment, set `save_instrument: true`. The shopper can also store credit cards during checkout. If you are using the [Checkout SDK](/stencil-docs/customizing-checkout/checkout-sdk), it can store the credit card as part of the checkout.
 
-```http title="Example request: Process payment and save credit card" lineNumbers
+```http filename="Example request: Process payment and save credit card" showLineNumbers
```

Comment:
```suggestion
```http filename="Example request: Process payment and save credit card" showLineNumbers copy
```